src/backend/distributed/commands/citus_add_local_table_to_metadata.c

```diff
@@ -1179,9 +1180,24 @@ DropDefaultExpressionsAndMoveOwnedSequenceOwnerships(Oid sourceRelationId,
 	Oid ownedSequenceId = InvalidOid;
 	forboth_ptr_oid(columnName, columnNameList, ownedSequenceId, ownedSequenceIdList)
 	{
-		DropDefaultColumnDefinition(sourceRelationId, columnName);
+		/*
```

Comment:
Why do we drop if it defaults to nextval? Maybe add -- with better wording: `We drop nextval expressions because Citus currently evaluates nextval on the shell table, not on the shards. Hence, there is no reason for keeping nextval. Also, distributed/reference table shards do not have - so be consistsnt with those` .